### ISSUE TITLE
A11y: [Start a call] Lock icon lacks alt text....

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5243]
- **Checkpoint ID:** 11.1.1.1b/12.2
- **View/Component:** Start a call

### Description
Lock icon lacks alt text.

### Recommendation
Add alt text like 'Encrypted call'.

--- 
*Report Date: 20.02.2026*
